src/tlo/methods/rti.py

```diff
@@ -4404,9 +4437,10 @@ def apply(self, person_id, squeeze_factor):
                 logger.debug(key='rti_general_message',
                              data=f"Tetanus vaccine requested for person {person_id} and given")
             else:
-                self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
-                logger.debug(key='rti_general_message',
-                             data=f"Tetanus vaccine requested for person {person_id}, not given")
+                if self._number_of_times_this_event_has_run < self._maximum_number_times_event_should_run:
+                    self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
+                    logger.debug(key='rti_general_message',
+                                 data=f"Tetanus vaccine requested for person {person_id}, not given")
```

Comment:
```suggestion
                logger.debug(key='rti_general_message', data=f"Tetanus vaccine requested for person {person_id}, not given")
```

Reply:
Looks logical. Will implement and test it